.docker/test-Publications.ini

```diff
@@ -54,6 +54,7 @@ ckan.datastore.default_fts_index_method = gist
 
 ## Site Settings.
 ckan.site_url = http://ckan:5000/
+WTF_CSRF_ENABLED = False
```

Comment:
Need a todo to fix plugins so this can be enabled (and not interfere with our own plugin)

Reply:
I don't think it's a plugin problem, actually, I think it's an issue on the group edit page. Will investigate more.